chrome/content/zotero/xpcom/utilities_internal.js

```diff
@@ -1660,6 +1660,16 @@ Zotero.Utilities.Internal.activate = new function() {
 	}
 };
 
+Zotero.Utilities.Internal.sendToBack = function() {
+	Zotero.Utilities.Internal.executeAppleScript(`
+		tell application "System Events"
+			if frontmost of application id "org.zotero.zotero" then
+				set visible of process "Zotero" to false
+			end if
+		end tell
+	`);
+}
+
```

Comment:
> In this version, using LibreOffice on macOS, if I go to add a citation it brings Zotero to the front and shows the Quick Format bar on top of that. If I switch back to master, the Quick Format bar shows on top of LibreOffice. (Zotero sometimes flashes to the front but it generally stays behind LibreOffice.)

Updated, but I'm not sure if we can rely on the stable name of the process "Zotero". Either way, this script should fail gracefully, so maybe not a big worry. This also fixes Zotero staying in the front completely, after interacting from Safari (which I was planning to address later, if there is enough interest/complaints from Safari gdocs users)